nautobot/utilities/templatetags/helpers.py

```diff
@@ -304,32 +304,27 @@ def percentage(x, y):
 
 @library.filter()
 @register.filter()
-def get_docs(model):
-    """Render and return documentation for the specified model.
+def get_docs_url(model):
+    """Return the documentation URL for the specified model.
+
+    Nautobot Core models have a path like docs/models/{app_label}/{model_name}
+    while plugins will have {app_label}/docs/models/{model_name}.
 
     Args:
         model (models.Model): Instance of a Django model
 
     Returns:
-        str: documentation for the specified model in Markdown format
+        str: URL for the documentation of the object.
 
     Example:
-        >>> get_docs(obj)
-        "some text"
+        >>> get_docs_url(obj)
+        "static/docs/models/dcim/site.html"
     """
-    path = "{}/models/{}/{}.md".format(settings.DOCS_ROOT, model._meta.app_label, model._meta.model_name)
-    try:
-        with open(path, encoding="utf-8") as docfile:
-            content = docfile.read()
-    except FileNotFoundError:
-        return "Unable to load documentation, file not found: {}".format(path)
-    except IOError:
-        return "Unable to load documentation, error reading file: {}".format(path)
-
-    # Render Markdown with the admonition extension
-    content = markdown(content, extensions=["admonition", "fenced_code", "tables"])
-
-    return mark_safe(content)
+    return (
+        f"{settings.STATIC_URL}{f'{model._meta.app_label}/' if model._meta.app_label in settings.PLUGINS else ''}"
+        f"docs/models/{f'{model._meta.app_label}/' if model._meta.app_label not in settings.PLUGINS else ''}"
+        f"{model._meta.model_name}.html"
```

Comment:
This is hard to read with the inline conditionals leading to inline strings. Can we do it as a basic `if model._meta.app_label in settings.PLUGINS: ... else: ...` instead?

Reply:
Sure, I changed this because Sorcery was complaining, but it would be much easier to read with two different return statements.